src/bin.js

```diff
@@ -52,39 +52,43 @@ var assureConfirmation = function(repos, cb) {
     }
   );
 };
-
-clean.get(
-  token,
-  {
-    progress: function(info) {
-      singleLineLog(
-        "Inspected " +
-          info.countInspected +
-          "/" +
-          info.totalToInspect +
-          (info.lastInspected ? " (" + info.lastInspected + ")" : "") +
-          "\n"
-      );
+try {
+  clean.get(
+    token,
+    {
+      progress: function(info) {
+        singleLineLog(
+          "Inspected " +
+            info.countInspected +
+            "/" +
+            info.totalToInspect +
+            (info.lastInspected ? " (" + info.lastInspected + ")" : "") +
+            "\n"
+        );
+      },
+      user: program.user,
+      warnings: function(msg, err) {
+        console.error(msg, err);
+      }
     },
-    user: program.user,
-    warnings: function(msg, err) {
-      console.error(msg, err);
-    }
-  },
-  function(err, repos) {
-    if (err) return abort(err);
-
-    if (!repos.length) {
-      console.log("No useless repositories found.");
-      process.exit(0);
-    }
+    function(err, repos) {
+      if (err) return abort(err);
 
-    assureConfirmation(repos, function() {
-      clean.remove(token, repos, function(errDeleting) {
-        if (errDeleting) return abort(errDeleting);
-        console.log("Done!");
+      if (!repos.length) {
+        console.log("No useless repositories found.");
         process.exit(0);
+      }
+
+      assureConfirmation(repos, function() {
+        clean.remove(token, repos, function(errDeleting) {
+          if (errDeleting) return abort(errDeleting);
+          console.log("Done!");
+          process.exit(0);
+        });
       });
-    });
-  }
-);
+    }
+  );
+} catch (error) {
+  console.error(error);
+  process.exit(1);
```

Comment:
Let’s reuse existing logic for displaying errors.

```suggestion
  abort(error);
```

Reply:
Definitely!